gem_tasks/cucumber.rake

```diff
@@ -0,0 +1,27 @@
+# Copied from https://github.com/cucumber/cucumber-ruby/blob/master/gem_tasks/cucumber.rake
+
+require 'cucumber/rake/task'
+require 'cucumber/platform'
+
+class Cucumber::Rake::Task
+  def set_profile_for_current_ruby
```

Comment:
I think it is important and if it is the way the gem recommends, it's probably the way to go.
My only concern is about our own team, since we always seem to forget to update CHANGELOG files, for example.

By the way, shouldn't there be a CHANGELOG entry for the changes made by this PR?